ir/shared/src/test/scala/org/scalajs/ir/PrintersTest.scala

```diff
@@ -937,6 +937,14 @@ class PrintersTest {
         CreateJSClass("Foo", List(ref("x", IntType), ref("y", AnyType))))
   }
 
+  @Test def printLinkTimeProperty(): Unit = {
+    assertPrintEquals(
+        """
+          |linkTimeProperty(foo)
```

Comment:
I guess the CI will fail on this one, but anyway:
```suggestion
          |<linkTimeProperty>(foo)
```

Reply:
Ow, nice catch, you're faster than Jenkins 😄